view/adminhtml/templates/boltpay/js/replacejs.phtml

```diff
@@ -0,0 +1,662 @@
+<?php
```

Comment:
Do we need this file at all?

Reply:
I plan to change this file to the component for admin panel.